sirepo/pkcli/service.py

```diff
@@ -141,8 +140,9 @@ def _cfg():
     if not __cfg:
         __cfg = pkconfig.init(
             ip=('0.0.0.0', _cfg_ip, 'what IP address to open'),
-            nginx_proxy_port=(8080, _cfg_int(5001, 32767), 'port on which nginx_proxy listens'),
-            port=(8000, _cfg_int(5001, 32767), 'port on which uwsgi or http listens'),
+            jupyterhub_port=(8005, _cfg_port, 'port on which jupyterhub listens'),
```

Comment:
81b4d8a8 look right? There was some weirdness getting the UwsgiClient to actually use the right port/ip since the changes are sent in env to sub-processes but weren't previously passed along to the UwsgiClient.